### DISCUSSION TITLE
How do I name cells?

### DISCUSSION THREAD

**Discussion by kalyan860** [andmarti1424/sc-im#716] [Ideas]
How do I assign variable names to cells like in excel or libreoffice?  

When using forumlas, it is not readable to see I7, C13 etc instead of readable names like price, discount etc?

**Comment by andmarti1424** [comment]
I am not aware of sc-im having this. nor in sc.

**Comment by kalyan860** [comment]
Can you please add this feature? It makes formulas more readable instead of typing B3 etc.
